init.lua

```diff
@@ -752,7 +752,7 @@ travelnet.on_receive_fields = function(pos, formname, fields, player)
    end
 
    if( not( fields.target )) then
-      minetest.chat_send_player(name, S("Please click on the target you want to travel to."));
+      minetest.chat_send_player(name, S("Choose a destination."));
```

Comment:
```suggestion
      minetest.chat_send_player(name, S("Choose a destination:"));
```

Reply:
This appears in the chat, not in the window. The choice of destinations doesn't appear below it, so a colon does not fit here. The message appears only after a non-destination is somehow selected. The string with a colon is in the GUI, and already has a colon in my branch: <https://github.com/poikilos/travelnet/blob/english-corrections/init.lua#L343>

Reply:
My bad! I didn't knew it was on the chat. Sorry 😋